exawind/NREL_5MW_Turbine/flexible/README.md

```diff
@@ -1,85 +1,87 @@
+- Hub-height streamwise velocity: 11.4 m/s
 - Hub-height flow direction: ~240.0 degrees
 
 <img src="../../../amr-wind/atmospheric_boundary_layer/convective_abl_nrel5mw/results/avgmetmast_0600.png" alt="10-Minute Average Virtual Met-mast Precursor Results" style="width:800px; background-color: #ffffff;"/>
 
 ## CFD Mesh
 
-<img src="figures_and_scripts/mesh-nrel5mw-abl.png" alt="Mesh layout" style="width:800px; background-color: #ffffff;"/>
+<img src="figures_and_scripts/split_mesh_layout.png" alt="Mesh layout" style="width:800px; background-color: #ffffff;"/>
 
-**Total number of cells: 177,729,426**
+**Total number of cells: 155,363,046**
 
-The near-body (Nalu-Wind) mesh was created using PGL, pyHyp, and Pointwise. Three blades are connected at the hub, and a tower is included.
+The near-body (Nalu-Wind) mesh was created using a proprietary surface mesher, pyHyp, and Pointwise. Three blades are split at the hub, and an unconnected tower is included.
 - Structured hex mesh
-- Cell count: 15,593,362
+- Cell count: 13,436,646
 
 Off-body (AMR-Wind) mesh was generated using the built-in capability of AMR-Wind. Off-body mesh information is summarized below 
 - Mesh topology: Cartesian with AMR
 - Domain in x= 0 to 5000m, y=0m to 5000m, z=0m to 1000m
 - Initial grid size: 10m
 - Finest cell size: 0.625m with 4 AMR levels
-- Cell count: 162,136,064
+- Cell count: 141,926,400
 
 ## Results
 
 <img src="figures_and_scripts/openfast_summary.png" alt="Summary of OpenFAST Results" style="width:800px; background-color: #ffffff;"/>
 
-Mean values over time period: 65s to 125s 
+#### Mean values over time period: 60s to 120s 
 
-- GenPwr: 4533.57 kW
-- GenTq: 39.78 kN-m
-- RotSpeed: 11.88 rpm
-- RotThrust: 805.17 kN
-- B1RootMxr: 1225828.64 N
-- B1RootMyr: 9651238.99 N
-- B1TipTDxr: 5.25 m
-- BldPitch1: 0.0 deg
+| variable | value |
+| -------- | ------- |
+|GenPwr|4551.89|
+|GenTq|39.34|
+|RotSpeed|12.07|
+|RotThrust|647.96|
+|B1RootMxr|770618.58|
+|B1RootMyr|7875054.77|
+|B1TipTDxr|4.20|
+|BldPitch1|2.80|
 
 
 ## Simulation Timings
 
-This benchmark was run on Sandia National Laboratories' machine "Flight" which is comprised of 112 Intel(R) Xeon(R) Platinum 8480+ CPU cores per node. 
+This benchmark was run at Sandia National Laboratories on a machine comprised of 112 Intel(R) Xeon(R) Platinum 8480+ CPU cores per node. 
 
 ### Processor Decomposition: 
 - Nalu-Wind: 672 ranks
-- AMR-Wind: 6496 ranks
+- AMR-Wind: 4928 ranks
 
 ### Timings:
-- Mean wall-clock time per timestep for entire simulation: 8.23s 
-- Mean wall-clock time per timestep per cell 4.628e-08s
+- Mean wall-clock time per timestep for entire simulation: 17.75s 
+- Mean wall-clock time per timestep per cell 1.142e-07s
+
+## Exawind Simulation Guide
+#### Step 1: Run the precursor for 5760 OpenFAST timesteps (approximately 1 rotation)
+srun -n1 openfastcpp inp.yaml 
+#### Step 2: Run the full Exawind suite
+srun -N 50 -n 5600 exawind --nwind 672 --awind 4928 nrel5mw.yaml &> log
```

Comment:
Recommend to add a code block around shell stuff, so it looks like
```bash
srun -N 50 -n 5600 exawind --nwind 672 --awind 4928 nrel5mw.yaml &> log
```
(renders prettier on the website)

Reply:
Code block should be in now